docs/photoniq/vwrs/vwrs-metrics.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 100
+title: Get VWRs Metrics
+---
+
+Macrometa PhotonIQ Virtual Waiting Rooms (VWRs) has built-in metrics that allow you to retrieve waiting room telemetry per waiting room, per minute. Data is stored for the past 90 days.
+
+Using VWRs metrics, you can:
+
+- Evaluate peak traffic flow through your waiting room and onto your site.
+- Determine how long users spent in the waiting room.
+- Use analytics to help calibrate your waiting room settings.
+
+## Get Metrics
+
+You can retrieve metrics using the following endpoints:
+
+- Get Metrics for a Domain - Retrieve the metrics for a specific domain. Set start and end times to get metrics for a specific time period.
+- Get All Metrics - Retrieve metrics for all domains. Set start and end times to get metrics for a specific time period.
+
+## Available Metrics
+
+The metrics response body includes the following properties:
+
+- **avg_waiting_time**: (integer) The average time users spend in the virtual waiting room before being granted access to the origin service (in milliseconds).
+- **domain_key**: (string) The domain key.
+- **duplicate_request_rate**: (integer) The percentage of duplicate requests detected and managed by the virtual waiting room service.
+- **origin_key**: (string) The origin key.
+- **peak_queue_length**: (integer) The highest number of users in the queue during a specific period.
+- **queue_abandonment_rate**: (integer) The percentage of users who leave the virtual waiting room without accessing the origin service.
+- **request_success_rate**: (integer) The percentage of requests to the origin service that are successfully processed after users leave the virtual waiting room.
+- **timestamp**: (number) The epoch time of the start interval when the metric was generated (in seconds).
+
+## Example Response
+
+```json
+[
+  {
+    "_key": "www.example1.com:2023-08-21_07:50:00",
+    "avg_waiting_time": 0,
+    "domain_key": "www.example1.com",
+    "duplicate_request_rate": 100,
+    "peak_queue_length": 100,
+    "queue_abandonment_rate": 0,
+    "request_success_rate": 100,
+     "usage_requests": 6000,
+    "timestamp": 1692604200000
```

Comment:
Strange.  The timestamp is in milliseconds.  I assume you copied this from somewhere.  We should fix this there too.
Remove the last three zeros from the timestamp to get it in seconds (as defined above).
You can verify the timestamp here: https://www.epochconverter.com/

All three samples have the timestamp in milliseconds.  Remove the last three zeros to get it to seconds for all of these.
@rahul-naik1 does our swagger file contain these examples?  If so, we should probably fix the timestamp.